### PR TITLE
feat: add popular collections table

### PR DESCRIPTION
**Pull request by shahin-hq**
## Summary
Closes: https://app.clickup.com/t/86dqnpmtg

This PR:
- adds the popular collections table
- adds pagination components
- adds filter components and logic (I am planning to extract filter logic into a hook so we can share it between collections overview and popular collections page)
- hooks up real data for the some parts of page

Hit http://127.0.0.1:8000/popular-collections to see the page 

## Checklist

-   [x] I checked my UI changes against the design and there are no notable differences
-   [x] I checked my UI changes for any responsiveness issues
-   [x] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I opened a corresponding card on Clickup for any remaining TODOs in my code
-   [ ] I added a short description on how to test this PR _(if necessary)_
-   [ ] I added a storybook entry for the component that was added _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [x] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

**Review by samharperpittam — CHANGES_REQUESTED** [review]
Trying to access this page as a guest results in 500 error.

File: app/Http/Controllers/PopularCollectionController.php [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            'title' => trans('metatags.popular-collections.title'),
+            'allowsGuests' => true,
+            'collections' => CollectionData::collection(
+                $collections->through(fn ($collection) => CollectionData::fromModel($collection, $user->currency()))
+            ),
+            'stats' => new CollectionStatsData(
+                nfts: 145,
+                collections: 25,
+                value: 256000,
+            ),
```
Comment (crnkovic) [inline app/Http/Controllers/PopularCollectionController.php:57]: How come this is hardcoded?

**Review by crnkovic — CHANGES_REQUESTED** [review]
Can you link to this page from "View all" link of "Popular Collections" on the `/collections`?

Please include all active filters when navigating with pagination.

Also, can you disable sorting on the table? Right now it seems as if you can click the headings to sort, but we aren't doing that for now.

**Comment by shahin-hq** [comment]
@samharperpittam should be good now, please check

**Comment by shahin-hq** [comment]
> Can you link to this page from "View all" link of "Popular Collections" on the /collections?
> Please include all active filters when navigating with pagination.
Also, can you disable sorting on the table? Right now it seems as if you can click the headings to sort, but we aren't doing that for now.

Should be good, please give a try

**Review by crnkovic — APPROVED** [review]

**Comment by crnkovic** [comment]
Pagination doesn't work but is fixed in the following PR: https://github.com/ArdentHQ/dashbrd/pull/555

**Review by samharperpittam — APPROVED** [review]

**Pull request merged**